src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Smart Bulletin No. 036"
+intro: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act)."
+number: "No. 036"
+order: 036
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Mar 10, 2023"
+lastupdated: "Feb 13, 2023"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| March 10, 2023 | Implementation of Smart Bulletin #36 |
+| February 13, 2023 | Removed broken link |
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+This Smart Bulletin provides operational guidance regarding Public Law 116-160, the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act. This statute requires the U.S. General Services Administration (GSA) to issue guidance on the use of charge cards to pay for charging federally-owned or managed Electric motor Vehicles (EVs) used for official Government business at commercial charging stations. Additionally, this bulletin provides guidance on the use of GSA SmartPay payment solutions for payment of rental car EV charging related to travel on official business. Note that unless otherwise indicated, the mention of specific commercial charging network names in this bulletin is for information purposes only, is not intended as an endorsement of any kind, and does not indicate a contractual relationship with GSA.
+
+## Summary
+
+Participating agencies/organizations use GSA SmartPay charge cards to obtain approximately $30 billion in products and services annually in support of their missions through approximately 100 million purchase, travel and fleet transactions. The CHARGE Act emphasizes the existing ability for agencies/organizations to use GSA SmartPay charge cards to pay for charging services for EVs at commercial charging facilities. At the time of this guidance being issued, GSA SmartPay contractor bank charge cards, including WEX and Voyager fleet cards, can be used to pay for charging services at ChargePoint, Inc. commercial charging stations. Other charging station networks currently only accept Visa & MasterCard branded cards through their respective payment apps and terminals. Certain EVgo chargers do include card readers for walk-up payment, but not all. As the electric vehicle charging industry continues to expand, the Center for Charge Card Management (CCCM) and our GSA SmartPay contractor banks will pursue additional relationships with charging station providers to increase access and acceptance at additional charging networks for federal agencies. Use of GSA SmartPay charge cards for both per transaction and charging subscriptions is permissible consistent with agency policy, applicable charge card spending limit(s), the Federal Acquisition Regulation and other applicable rules/regulations. 
```

Comment:
Participating agencies/organizations use GSA SmartPay charge cards to obtain approximately $30 billion in products and services annually in support of their missions through approximately 100 million purchase, travel, and fleet transactions. The CHARGE Act emphasizes the existing ability for agencies/organizations to use GSA SmartPay charge cards to pay for charging services for EVs at commercial charging facilities. When this guidance is issued, GSA SmartPay contractor bank charge cards, including WEX and Voyager fleet cards, can be used to pay for charging services at ChargePoint, Inc. commercial charging stations. Other charging station networks only accept Visa & MasterCard branded cards through their respective payment apps and terminals. Specific EVgo chargers include card readers for walk-up payments, but not all. As the electric vehicle charging industry expands, the Center for Charge Card Management (CCCM) and our GSA SmartPay contractor banks will pursue additional relationships with charging station providers to increase access and acceptance at additional charging networks for federal agencies. GSA SmartPay charge cards per transaction and charging subscriptions are permissible and consistent with agency policy, applicable charge card spending limit(s), the Federal Acquisition Regulation, and other applicable rules/regulations.